scene/resources/visual_shader.h

```diff
@@ -479,10 +479,10 @@ class VisualShaderNodeGroupBase : public VisualShaderNode {
 	void _apply_port_changes();
 
 protected:
-	Vector2 size;
-	String inputs;
-	String outputs;
-	bool editable;
+	Vector2 size = Size2(0, 0);
+	String inputs = "";
+	String outputs = "";
```

Comment:
`Vector2` is default initialized to `Vector2(0, 0)`, and similarly `String` defaults to `""`, so you don't need to make this explicit.

Reply:
Yeah, I just remember some cases when the compiler ignores it and pushes uninitialized non-zero data instead.